clarifai/client/auth/helper.py

```diff
@@ -5,11 +5,15 @@
 
 from clarifai_grpc.channel.clarifai_channel import ClarifaiChannel
 from clarifai_grpc.grpc.api import resources_pb2, service_pb2_grpc
+
+from clarifai import __version__
 from clarifai.utils.constants import CLARIFAI_PAT_ENV_VAR, CLARIFAI_SESSION_TOKEN_ENV_VAR
 
 DEFAULT_BASE = "https://api.clarifai.com"
 DEFAULT_UI = "https://clarifai.com"
 
+REQUEST_ID_PREFIX = "sdk-python-" + str(__version__)
```

Comment:
Since `__version__` is already a str, it can be

```
REQUEST_ID_PREFIX = f"sdk-python-{__version__}"
```

Reply:
Let me know if it needs to change

Reply:
Done

Reply:
> Note: version must be in format: major.minor

@srikanthbachala20 did you test with full semantic version(`major.minor.patch`) and it works? i.e. Is this trimmed automatically by BE? 

If it doesn't work, please also trim it to `major.minor` instead of default `major.minor.patch`:

```
REQUEST_ID_PREFIX = f"sdk-python-{'.'.join(__version__.split('.')[:2])}"
```

Reply:
Yes, I have tested with full semantic version. It is trimmed automatically by BE